data/categories.txt

```diff
@@ -14789,6 +14828,46 @@ tr:İskele|Marina|Rıhtım
 uk:5Пристань|3Марина
 zh-Hans:游艇码头
 
+piste:type
+en:3Skiing|Sledding|Ski run
```

Comment:
"3Skiing"?

Reply:
3 is a prefix chars number. After entering "Ski" the app will suggest "Skiing" category.